tools/python/boutiques/schema/examples/tests.json

```diff
@@ -0,0 +1,473 @@
+{
```

Comment:
I'm a bit confused - how come this is a new file rather than just adding tests to the `good.json` file that we already had and which most of this was copied from?

Reply:
Yes that's true. I can c/p the tests in tests_good.json (that are meant to pass successfully) to good.json

However know that the other test_failure* JSON files are technically correct, they just fail because the test cases do not pass. Do you still want to proceed with that information ?

BTW disregard tests.json, this file should have been deleted.

Reply:
OK perfect, the only one I didn't love was `tests.json` :)

Reply:
@Trajan64 should this file, `tests.json` still have been deleted?